src/main/java/seedu/address/model/person/Role.java

```diff
@@ -0,0 +1,21 @@
+package seedu.address.model.person;
+
+public class Role {
+    private final String role;
+    public static final String MESSAGE_CONSTRAINTS =
+            "Roles should only contain alphanumeric characters and spaces, and it should not be blank";
+    
+    public static final String VALIDATION_REGEX = "[\\p{Alnum}][\\p{Alnum} ]*";
+
+    public Role(String role) {
+        this.role = role;
+    }
+
+    public String getRole() {
+        return this.role;
+    }
```

Comment:
We don't need a getter here.

Also, it is already inside `Role` class so we can use `get` instead. This is like a more common API usage. `person.getRole().get()` is less repetitive than `person.getRole().getRole()`